script.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
-cd /lib 
+cd /lib
+ln -s /lib64/* /lib
```

Comment:
This would break applications designed for /lib32.
Can you specify which specific files to link?

Reply:
Linking ld-linux-x86-64.so.2 to the lib/ location.

Reply:
Awesome, please update the pull request accordingly.
Thanks.